website/blog/2021-06-24-introduce-superbench.md

```diff
@@ -8,4 +8,113 @@ author_url: https://github.com/TobeyQin
 tags: [superbench, announcement]
 ---
 
-We are very happy to introduce [SuperBench](https://github.com/microsoft/superbenchmark) to help you validate AI infrastructure.
+
+This blog is to introduce [SuperBench](https://github.com/microsoft/superbenchmark) to help you validate AI infrastructure.
+
+We designed and developed SuperBench as a avalidation and profiling tool for AI infrastructure. 
+We try our best to create a modular, easy-to-use, standardized, and extensible tool. 
+Help users to complete and accurate assessment of AI infrastructure, and bring good user experience.
+
+
+## The Advantages of SuperBench
+
+### Easy-to-use CLI
+
+In order to achieve one-click deployment, one-click user experience, SuperBench provides a command line interface to help users deploy and run benchmarks. 
+Empowered by SuperBench CLI, user can deploy and run their benchmarks with only one command, which greatly shorten the learning time of using tools, 
+help user easily evaluate the performance of AI workload.
+
+Below is a simple example to show how to deploy and run benchmarks locally. For more information, 
+please view [CLI Document](https://github.com/microsoft/superbenchmark/blob/main/docs/cli.md)
+
+1. Deploy
+
+    ```bash
+    sb deploy -f local.ini
+    ```
+2. Run Benchmark
+
+    ```bash
+    sb run -f local.ini -c config.yaml
+    ```
+
+Among them, `local.ini` is the configuration file to manage nodes. If you want to run benchmarks locally, the node is `localhost`.
```

Comment:
Actually, I cannot understand the meaning.

Reply:
Changed to:
Among them, `local.ini` is the configuration file to manage worker nodes that will actually run benchmarks.
In below case, the worker node is `localhost`, same as control node.